### DISCUSSION TITLE
[glfw error 65544]: process_desktop_settings: failed with error... everytime I open kitty

### DISCUSSION THREAD

**Discussion by charbelnicolas** [kovidgoyal/kitty#6975] [Q&A]
Hello, I get the following error every time I open kitty:

`[glfw error 65544]: process_desktop_settings: failed with error: [org.freedesktop.DBus.Error.UnknownMethod] No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop`

Is there a fix for it?

**Comment by kovidgoyal** [comment]
Install a desktop settings portal.

On Sat, Jan 06, 2024 at 04:14:16PM -0800, Charbel Nicolas wrote:
> Hello, I get the following error every time I open kitty:
> 
> `[glfw error 65544]: process_desktop_settings: failed with error: [org.freedesktop.DBus.Error.UnknownMethod] No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop`
> 
> Is there a fix for it?
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6975
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by charbelnicolas** [reply]
Thank you but I don't think labwc or wlroots based desktop have a desktop settings portal. I know gnome and kde do though.

**Comment by kovidgoyal** [comment]
On Sat, Jan 06, 2024 at 07:12:28PM -0800, Charbel Nicolas wrote:
> Thank you but I don't think labwc or wlroots based desktop have a desktop settings portal. I know gnome and kde do though.

Sure they do xdg-desktop-portal-wlr for instance. And if you dont want
to install aportal ignore the error.

**Reply by charbelnicolas** [reply]
Hmm I already have xdg-desktop-portal-wlr installed, maybe I haven't enabled it properly. I'll check that out, thank you.

**Reply by Baal721** [reply]
hey, did you manage to solve this?
